### PR TITLE
MGDAPI-1996 - change to support RHOAM MT directory rename

### PR DESCRIPTION
**Pull request by KevFan**
## Description
Change to support manged tenants directory name change for RHOAM once https://gitlab.cee.redhat.com/service/managed-tenants/-/merge_requests/1008 is merged 

Jira:
* https://issues.redhat.com/browse/MGDAPI-1996

## Verification 
* Passing unit tests in PR should be sufficient

### CODE REVIEW

File: cmd/getSupportedVersions_test.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ func TestGetSupportedVersionsCmd(t *testing.T) {
 			olmType:       "managed-api-service",
 			majorVersions: 1,
 			minorVersions: 3,
-			repo:          "https://gitlab.cee.redhat.com/service/managed-tenants.git",
+			repo:          "https://gitlab.cee.redhat.com/chfan/managed-tenants.git",
```
Comment (KevFan) [inline cmd/getSupportedVersions_test.go:28]: Currently just pointing to own fork with the directory name change already applied. 

Will be updated back once MT MR is merged https://gitlab.cee.redhat.com/service/managed-tenants/-/merge_requests/1008

**Comment by austincunningham** [comment]
/assign @laurafitzgerald  @briangallagher 
Can you approve ?

**Pull request merged**